### DISCUSSION TITLE
Getting an error in Fooocus while on google Colab

### DISCUSSION THREAD

**Discussion by Airewolf** [lllyasviel/Fooocus#1767] [Q&A] [answered]
i am getting an error just as the image starts processing in google colab.
The end dialog is ^C.
![image](https://github.com/lllyasviel/Fooocus/assets/140367114/fc94c1e8-acf0-40ce-937d-15e9f345b086)

**Comment by omgbaby** [comment]
I'm suddenly getting connection errors today with colab. Coincidence?

**Comment by Jpru20** [comment]
I'm paying for collab pro and I get a runtime error each time I'm trying to run an image prompt after I create the orginal image. What am I doing wrong?

**Comment by eddyizm** [comment, marked as answer]
I get that error when I max out the system ram, usually trying to use too large of an image prompt or two images. 
Check your resources and see if you are redlining.

**Reply by GodFazer** [reply]
just faceswap, even without prompt - crash
